### ISSUE TITLE
Feature Request: GRUBlockFusedCell in tf.contrib.rnn

### ISSUE THREAD

**Issue by dy-octa** [tensorflow/tensorflow#22460]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Linux Ubuntu 16.04
- **Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device**: /  
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: 1.10.0
- **Python version**: 3.6.0
- **Bazel version (if compiling from source)**: /
- **GCC/Compiler version (if compiling from source)**: /
- **CUDA/cuDNN version**: /  
- **GPU model and memory**: /  
- **Exact command to reproduce**: /  

### Describe the problem
Fused RNN cells provide much better performance. Currently only tf.contrib.rnn.LSTMBlockFusedCell is provided. More, tf.contrib.cudnn_rnn.CudnnGRU, which has similar performance with typical FusedCell, is incompatible with normal GRUCell due to differences in computing mechanisms, hence pretrained GRU parameters cannot be used by CudnnGRU. So I wonder if a fused version of GRUCell could be implemented.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Bazel version

**Comment by tensorflowbutler** [comment]
It has been 15 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 29 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 44 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
We are closing this issue for now due to lack of activity. Please comment if this is still an issue for you. Thanks!
